mrp_production_category_notes/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2025 Ángel Rivas <angel.rivas@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "MRP Production Category Notes",
+    "version": "15.0.1.0.0",
+    "category": "Manufacturing",
+    "summary": (
+        "This module adds a new field to product categories "
+        "that allows adding notes to mrp orders."
+    ),
+    "website": "https://github.com/sygel-technology/sy-manufacture",
+    "author": "Sygel, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["mrp", "mrp_production_note"],
```

Comment:
You don't need mrp dependency, as you already have mrp_production_note dependency.